analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -183,6 +185,31 @@ open class Npm(
         }
     }
 
+    /**
+     * Parse information about the author. According to
+     * https://docs.npmjs.com/files/package.json#people-fields-author-contributors, there are two formats to
+     * specify the author of a package: An object with multiple properties or a single string.
+     */
+    private fun parseAuthor(json: JsonNode): SortedSet<String> =
```

Comment:
Shouldn't this better simply return a (nullable) `String` as the set only ever contains at most a single element?

Reply:
For the calling code it is more convenient that this function returns a set because this is needed for the package metadata. However, it might then be more suitable to rename the function to `parseAuthors()`. WDYT?

Reply:
When keeping the implementation as-is then yes, we should rename the function to plural `parseAuthors()`. But I'm still wondering whether instead writing `sortedSetOf(parseAuthor())` at each call site is really that bad.

Maybe a (quite academic) argument for using plural `parseAuthors()` would be that we then could easily extend the function if NPM would start to support author arrays, similar to like we already do the parsing in `parseLicenses()`.

I'll leave the decision up to you.

Reply:
I went for `parseAuthors()`. Writing `sortedSetOf(parseAuthor())` on the caller side would not be sufficient because the case of a *null* result still has to be handled.

Reply:
Right. While there is `setOfNotNull()`, there's unfortunately not `sortedSetOfNotNull()`.